src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,140 +1,121 @@
+import Tippy from '@tippyjs/react';
 import clsx from 'clsx';
-import React, {
-  ReactNode,
-  useState,
-  useRef,
-  useEffect,
-  MutableRefObject,
-} from 'react';
+import * as React from 'react';
 import styles from './Tooltip.module.css';
-import { ESCAPE_KEYCODE, TAB_KEYCODE } from '../../util/keycodes';
-import { Button } from '../Button/Button';
 
-export interface Props {
+// Full list of Tippy props: https://atomiks.github.io/tippyjs/v6/all-props/
+type TooltipProps = {
   /**
-   * Alignment variations
-   * - **right** yields a tooltip that appears to the right of the trigger
-   * - **below** yields a tooltip that appears below the trigger
-   * - **left** yields a tooltip that appears to the left of the trigger
+   * The trigger element the tooltip appears next to.
    */
-  align?: 'right' | 'below' | 'left';
+  children?: React.ReactElement;
   /**
-   * The visually-hidden button text for the tooltip trigger button
+   * The trigger element the tooltip appears next to.
+   *
+   * Use this instead of `children` if the trigger element is being
+   * stored in a ref. Most cases will use `children` and not
+   * `reference`.
    */
-  buttonText?: string;
```

Comment:
I can't figure out what this prop was 🤔 Do you understand the comment?

Reply:
Uhh not sure what the comment means but the b/m tooltip also exported the trigger button, and `buttonText` is for the text inside the trigger button 
![image](https://user-images.githubusercontent.com/86632227/157307477-479bb8d8-8d88-45a0-b26e-306f1566ad02.png)
![Screen Shot 2022-03-08 at 11 06 04 AM](https://user-images.githubusercontent.com/86632227/157309230-3a71498f-3147-4366-944e-88f08540e65f.png)